lib/traject/config/folio_config.rb

```diff
@@ -1649,7 +1648,7 @@ def accumulate_summary_struct_fields(matching_fields, tag, label, marc, accumula
   end
 
   accumulator << { label:, fields:,
-                   unmatched_vernacular: unmatched_vern } unless fields.empty? && unmatched_vern.nil?
+                   unmatched_vernacular: unmatched_vern } unless fields.empty? && unmatched_vern.empty?
```

Comment:
The `unless` is pretty confusing. Is this the same thing?

```suggestion
                   unmatched_vernacular: unmatched_vern } if fields.empty? && !unmatched_vern.empty?
```